docs/src/stratospheric_reaction_benchmark.md

```diff
@@ -0,0 +1,306 @@
+algs1 = [MPRK22(1.0, small_constant = 1e-6); MPRK43I(0.5, 0.75)]
+labels1 = ["MPRK22(1.0, sc=1e-6)"; "MPRK43I(0.5,0.75)"]
+
+# select OrdinaryDiffEq methods
+algs2 = [TRBDF2(); Kvaerno3(); KenCarp3(); Rodas3(); ROS2(); ROS3(); Rosenbrock23()]
+labels2 = ["TRBDF2"; "Kvearno3"; "KenCarp3"; "Rodas3"; "ROS2"; "ROS3"; "Rosenbrock23"]
+
+# compute work-precision data
+wp = work_precision_adaptive(prob, algs1, labels1, abstols, reltols, alg_ref; compute_error)
+work_precision_adaptive!(wp, prob, algs2, labels2, abstols, reltols, alg_ref; compute_error, 
+                        isoutofdomain = isnegative)
+
+# plot work-precision diagram
+plot(wp, [labels1; labels2]; title = "Stratospheric reaction benchmark", legend = :bottomleft,     
+     color = permutedims([1, 3, repeat([4], 3)..., repeat([5], 4)...]),
+     xlims = (10^-8, 10^0), xticks = 10.0 .^ (-8:1:0),
+     ylims = (2*10^-4, 5*10^0), yticks = 10.0 .^ (-5:1:0), minorticks = 10)
+```
+
+We see that MPRK methods are advantageous if low accuracy is acceptable.
+
+In addition,  we compare `MPRK22(1.0, small_constant = 1e-6)` and  `MPRK43I(0.5, 0.75)` to some [recommended solvers](https://docs.sciml.ai/DiffEqDocs/dev/solvers/ode_solve/) of higher order from [OrdinaryDiffEq.jl](https://docs.sciml.ai/OrdinaryDiffEq/stable/). Again, to guarantee positive solutions we select the solver option `isoutofdomain = isnegative`.
+
+```@example stratreac
+# select OrdinaryDiffEq methods
+algs3 = [Rodas5P(); Rodas4P(); RadauIIA5()]
+labels3 = ["Rodas5P"; "Rodas4P"; "RadauIIA5"]
+
+# compute work-precision data
+wp = work_precision_adaptive(prob, algs1, labels1, abstols, reltols, alg_ref; compute_error)
+work_precision_adaptive!(wp, prob, algs3, labels3, abstols, reltols, alg_ref; compute_error, 
+                        isoutofdomain = isnegative)
+
+# plot work-precision diagram
+plot(wp, [labels1; labels3]; title = "Stratospheric reaction benchmark", legend = :topright,     
+     color = permutedims([1, 3, repeat([4], 3)...]),
+     xlims = (10^-7, 10^0), xticks = 10.0 .^ (-8:1:0),
+     ylims = (2*10^-4, 5*10^0), yticks = 10.0 .^ (-5:1:0), minorticks = 10)
+```
+
+Again, it can be seen that MPRK methods are only advantageous if low accuracy is acceptable.
+
+### Fixed time steps sizes
+
+Here we use fixed time step sizes instead of adaptive time stepping.
+We use the functions [`work_precision_fixed`](@ref) and [`work_precision_fixed!`](@ref) to compute the data for the diagrams.
+Please note that these functions set error and computing time to `Inf`, whenever a solution contains negative elements. 
+Consequently, such cases are not visible in the work-precision diagrams.
+
+Within the work-precision diagrams we use the following time step sizes.
+
+```@example stratreac
+# set time step sizes
+dt0 = 48 * 60 # 48 minutes
+dts = dt0 ./ 2.0 .^ (0:1:10)
+nothing # hide
+```
+
+Other than for adaptive schemes increasing `small_constant` has no positive effect, as the following examples show.
```

Comment:
There is no case with `small_constant = 1e-6` as above. Why? This is a bit confusing